__tests__/api.test.js

```diff
@@ -1,15 +1,16 @@
 const request = require("supertest");
+const db = require("../db/connection.js");
 const seed = require("../db/seeds/seed.js");
-const { server, app } = require("../main_app");
+const { app } = require("../main_app");
 const destroy = require("../__helpers__/helpers.testing.js");
```

Comment:
We can get rid of destroy now if it isn't being used

Reply:
fixed :)